evasquez26/angular4-startup
# S5-ONE-95 Create organization

## Story / Task

see [ONE-95](https://bixlabs.atlassian.net/browse/ONE-95)


## What this PR does?

Adds functionality to add a request to create an organization

### Before 

You could not create an organization
### After

You have the option to create an organization

### How to test / reproduce

- Login to the app
- Click "schools" in the left bar
- Click New organization
- Fill the form
- Click save
- You should see a success message

## Migrations

NO


Thank you.
